scss/_patterns_lists.scss

```diff
@@ -133,11 +119,21 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
     .p-list__item {
       @extend %vf-list-item;
       @include vf-list-item-divided;
+
+      &.is-ticked {
+        background-position-y: 0.85rem;
+      }
     }
 
     &.is-split .p-list__item:last-of-type {
       border-bottom: 1px solid $color-mid-light;
     }
+
+    ul,
```

Comment:
Looking at HTML code it seems that nested lists also have a class `p-list--divided`, so we could target class name here instead of element to avoid element selector.

Would the component even work if we nest a list without this class name? Or with different list class name?

Reply:
Targetting the classname would work in the example because both lists are `p-list--divided` but let's say in the wild someone uses a different style for the inner list. It would not be indented. this is why I targeted the tag instead, as a failsafe. What do you think?